### PR TITLE
Fixups for (#1691)

### PR DESCRIPTION
**Pull request by xbrianh**
A few missed adaptations to use SQS instead of SNS

### CODE REVIEW

File: daemons/dss-index/app.py [hunk 1]
```diff
@@ -33,16 +33,18 @@ def dispatch_s3_indexer_event(event, context) -> None:
     if event.get("Event") == "s3:TestEvent":
         logger.info("DSS index daemon received S3 test event")
     else:
-        _handle_event(Replica.aws, event, context)
+        for event_record in event["Records"]:
```
Comment (Bento007) [inline daemons/dss-index/app.py]: Why was this change needed?

Comment (Bento007) [reply daemons/dss-index/app.py]: will indexer receive multiple records to process in one request?

Comment (xbrianh) [reply daemons/dss-index/app.py]: SQS->Lambda sends messages in batches:
https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html

Comment (Bento007) [reply daemons/dss-index/app.py]: From https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html
> All messages in a failed batch return to the queue, so your function code must be able to process the same message multiple times without side effects.

Will this be a problem?

Comment (xbrianh) [reply daemons/dss-index/app.py]: Not as long as the indexer is idempotent, which I believe it is.

Also, if needed, batch size may be configured.

Comment (Bento007) [reply daemons/dss-index/app.py]: adding idempotent test for indexer #1697

Comment (Bento007) [reply daemons/dss-index/app.py]: Fixes the failing tests and i'll review.
